`make` fails

```
stephen at Stephens-MacBook-Pro in ~/repos
$ git clone git@github.com:yorkie/serve.git ./github.com/yorkie/serve
Cloning into './github.com/yorkie/serve'...
remote: Counting objects: 37, done.
remote: Compressing objects: 100% (28/28), done.
remote: Total 37 (delta 15), reused 30 (delta 8), pack-reused 0
Receiving objects: 100% (37/37), 149.74 KiB | 0 bytes/s, done.
Resolving deltas: 100% (15/15), done.
Checking connectivity... done.

stephen at Stephens-MacBook-Pro in ~/repos
$ cd github.com/yorkie/serve/

stephen at Stephens-MacBook-Pro in ~/repos/github.com/yorkie/serve on master
$ make
rm -rf *.gc* *.dSYM *.exe *.obj *.o a.out serve
cc serve.c mongoose.c -o dest/serve -W -Wall -pthread 
ld: can't open output file for writing: dest/serve, errno=2 for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [serve] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -13,6 +13,7 @@ endif
 all: clean $(PROG)
 
 $(PROG): $(SOURCES)
+	@mkdir -p dest
 	$(CC) $(SOURCES) -o dest/$@ $(CFLAGS)
 
 $(PROG).exe: $(SOURCES)

